### PR TITLE
[40856]Team planner animation

### PR DESCRIPTION
**Pull request by bsatarnejad**
Add animation of team planner to the source and update common upsale template and all the places we are using it.

https://community.openproject.org/work_packages/40856/activity

### CODE REVIEW

File: modules/team_planner/app/views/team_planner/team_planner/upsale.html.erb [hunk 1]
```diff
@@ -5,5 +5,6 @@
                feature_title: t('team_planner.upsale.title'),
                feature_description: t('team_planner.upsale.description'),
                feature_reference: 'team_planner',
-               feature_image: 'enterprise/team-planner.png'
+               feature_image: 'enterprise/team-planner.png',
```
Comment (HDinger) [inline modules/team_planner/app/views/team_planner/team_planner/upsale.html.erb]: The image was only a placeholder and can be removed now.

**Review by HDinger — CHANGES_REQUESTED** [review]
It works locally, but it won't on production. Please move the video to the assets folder as `public` is not part of the asset pipeline and will thus not be shown on production.

One other thing: The upsale template for other places than the team planner now look a bit weird. For one, the image is blurry. I think it needs to be smaller or we need an image with a higher resolution. 
And second, since it is centred, it looks like the headline is floating in the middle of the page. I'd suggest to move the whole template to the top.
Bildschirmfoto 2022-03-03 um 08 20 51

File: modules/team_planner/app/views/team_planner/team_planner/upsale.html.erb (hunk 1 above)
Comment (bsatarnejad) [reply modules/team_planner/app/views/team_planner/team_planner/upsale.html.erb]: Hi @HDinger As I understood In Rails we should add videos to the public folder, https://apidock.com/rails/v5.2.3/ActionView/Helpers/AssetTagHelper/video_tag I have tested it by adding the video to assets folder, but it was not displayed on the page. Is there another way to show a video in this case?

Can we have a higher resolution image for other places? Should I contact product team for it?

Comment (HDinger) [reply modules/team_planner/app/views/team_planner/team_planner/upsale.html.erb]: @bsatarnejad In the documentation it says "The sources can be full paths or files that exist in your public videos directory." For our case we need the full path option. On production, images and videos needs be processed by the asset pipeline, in order to display them correctly. As I said, the `public` folder is not included in that pipeline resulting in the video not being found.
I just tested it myself and it worked correctly. Be sure to add the video to `openproject/app/assets/videos` and **not** to the frontend folder. Further you need to restart your server once you did that.

File: frontend/src/global_styles/content/_enterprise.sass [hunk 2]
```diff
@@ -15,6 +11,12 @@
     margin-bottom: 20px
     border: 1px solid var(--primary-color-dark)
     border-radius: 20px
+    
+  .widget-box--teaser-video
+    width: 100%
+    margin-bottom: 20px
+    border: 1px solid var(--primary-color-dark)
```
Comment (b12f) [inline frontend/src/global_styles/content/_enterprise.sass]: `border: 1px solid $spot-color-main-dark`

**Review by HDinger — APPROVED** [review]
👍

**Pull request merged**